Frameworks/include/CFFoundationInternal.h

```diff
@@ -1,6 +1,6 @@
 //******************************************************************************
 //
-// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
These clean-ups, though nice, seem unrelated to this PR.  Just revert these files?